source/Octopus.Server.Client/LocationChecker.cs

```diff
@@ -18,13 +16,14 @@ public static void CheckAssemblyLocation()
             {
                 if (octopusRegNode == null) 
                     return;
-                CheckBasedOnApplication(octopusRegNode, "Server");
-                CheckBasedOnApplication(octopusRegNode, "Tentacle");
+                // CheckBasedOnApplication(octopusRegNode, "Server");
```

Comment:
This is being removed as this part of the code only runs in the .NET Framework build of Octopus.Client and we don't ship that with server since we converted to .NET.

... Or do we, I'll follow up.